firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3410,6 +3410,38 @@ extension String {
         comment: "The label for the URL field when editing a bookmark")
 }
 
+// MARK: - Root Bookmarks Folder Empty State
+extension String {
+    public struct RootBookmarksFolderEmptyState {
+        public static let Title = MZLocalizedString(
+            key: "RootBookmarksFolderEmptyState.Title.v135",
+            tableName: "RootBookmarksFolderEmptyState",
+            value: "No bookmarks yet",
+            comment: "Title for the empty state of the root bookmarks folder when it does not contain any items")
```

Comment:
These comments should be a bit more descriptive for the translators. Something like, "In the bookmarks library, at the top level of the bookmarks library, the title for the placeholder screen that shows when there are no already saved bookmarks yet."